Source/main.cpp

```diff
@@ -26,13 +26,7 @@ int main(int argc, char* argv[])
     MPI_Init_thread(&argc, &argv, MPI_THREAD_MULTIPLE, &provided);
     AMREX_ALWAYS_ASSERT(provided >= MPI_THREAD_MULTIPLE);
 #   else
```

Comment:
When do we want this again, currently only for async I/O?
Because then we might want to add it to AMReX' GNUmake options, too and make it optional for CMake with MPI as well (there we already have an option in AMReX that we can wrap).